src/components/MoneyRequestHeader.tsx

```diff
@@ -147,7 +156,11 @@ function MoneyRequestHeader({
             return {title: getStatusIcon(Expensicons.ReceiptScan), description: translate('iou.receiptScanInProgressDescription'), shouldShowBorderBottom: true};
         }
         if (TransactionUtils.hasPendingRTERViolation(TransactionUtils.getTransactionViolations(transaction?.transactionID ?? '', transactionViolations))) {
-            return {title: getStatusIcon(Expensicons.Hourglass), description: translate('iou.pendingMatchWithCreditCardDescription'), shouldShowBorderBottom: true};
+            return {
```

Comment:
This should be moved above the `isScanning` message. That is how it is in oneTransactionReports. When there is an RTER violation that message is shown instead of the 'being scanned' message.